tests/postman/README.md

```diff
@@ -0,0 +1,129 @@
+# Test Examples
+
+This folder contains a newman/postman collection and sample input data required to test issuance of credentials from the traceability vocab against implementations of the VC-API as defined in the traceability interop profile
+
+to execute examples from CLI:
+
+```shell
+newman run ./collections/reference/vc-http-api-supply-chain.json -d ./data/reference/vc-http-api-supply-chain-credentials.json 
+```
+
+A variation of this test set will be generalized and extended out to be incorporated into CI/CD for the overall traceability specification objects
+
+## Core Profile and Interop Tests
+
+All APIs and configs will be tested with Newman/Postman, and additional settings (security, etc.) may be tested with other core system utilities and/or JS
+
+This package may be executed with the following:
+
+```bash
+node .
+```
+
+Which will execute all configured tests via node
+
+To get help on running the tests, execute:
+
+```bash
+bash$ node . -h
+Traceability Interop Testing
+Usage: trace-interop-test [options]
+
+Options:
+  -V, --version                  output the version number
+  -s, --service <file>           use the specified service provider test collection (default: "./collections/service-providers.json")
+  -r, --reference <file>         use the specified reference VC test collection (default: "./collections/reference-credentials.json")
+  -sd, --servicedata <file>      use the specified service provider data collection (default: "./data/service-providers.json")
+  -rd, --referencedata <file>    use the specified reference VC data collection (default: "./data/reference-credentials.json")
+  -rd, --reportdir <folder>      use the specified service provider data collection (default: "./newman")
+  -t, --tests <all...>           use the specified tests, "none" is provided as an option for dev purposes (choices: "all", "service", "reference", "interop", "none", default: ["all"])
+  -d, --dids <key, web, all...>  use the specified did methods (default: ["key"])
+  -n, --names <all...>           test only the service provider as identified by name (default: ["all"])
```

Comment:
This probably means changing the executable's source, wherever that is. It's to 
make the `-h` output more readable on typical terminal screens
```suggestion
  -s, --service <file>           use the specified service provider test collection 
                                 (default: "./collections/service-providers.json")
  -r, --reference <file>         use the specified reference VC test collection 
                                 (default: "./collections/reference-credentials.json")
  -sd, --servicedata <file>      use the specified service provider data collection 
                                 (default: "./data/service-providers.json")
  -rd, --referencedata <file>    use the specified reference VC data collection 
                                 (default: "./data/reference-credentials.json")
  -rd, --reportdir <folder>      use the specified service provider data collection 
                                 (default: "./newman")
  -t, --tests <all...>           use the specified tests, "none" is provided as an 
                                 option for dev purposes (choices: "all", "service", 
                                 "reference", "interop", "none", default: ["all"])
  -d, --dids <key, web, all...>  use the specified did methods (default: ["key"])
  -n, --names <all...>           test only the service provider as identified by name 
                                 (default: ["all"])
```

Reply:
I'll file an upstream issue with the cli lib - good catch - definitely better ways to optimize the output